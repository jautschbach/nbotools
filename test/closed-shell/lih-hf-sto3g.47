 $GENNBO  NATOMS=2  NBAS=6  UPPER  BODM  FORMAT=PRECISE  $END
 $NBO plot FILE=lih-hf-sto3g  $END
 $COORD
 ORCA Job: lih-hf-sto3g
     3    3       0.925040       0.199362       0.000000
     1    1      -0.604960       0.199362       0.000000
 $END
 $BASIS
  CENTER =     1     1     1     1     1     2
   LABEL =     1     1   103   101   102     1
 $END
 $CONTRACT
  NSHELL =     4
    NEXP =    12
   NCOMP =     1     1     3     1
   NPRIM =     3     3     3     3
    NPTR =     1     4     7    10
     EXP =  0.161195750000E+02  0.293620070000E+01  0.794650500000E+00
            0.636289700000E+00  0.147860100000E+00  0.480887000000E-01
            0.636289700000E+00  0.147860100000E+00  0.480887000000E-01
            0.342525091000E+01  0.623913730000E+00  0.168855400000E+00
      CS =  0.884856287789E+00  0.855794457874E+00  0.266713752926E+00
           -0.507585163759E-01  0.678936361973E-01  0.512403447492E-01
            0.000000000000E+00  0.000000000000E+00  0.000000000000E+00
            0.276934360953E+00  0.267838851753E+00  0.834736696169E-01
      CP =  0.000000000000E+00  0.000000000000E+00  0.000000000000E+00
            0.000000000000E+00  0.000000000000E+00  0.000000000000E+00
            0.126299062547E+00  0.794202811582E-01  0.125815088415E-01
            0.000000000000E+00  0.000000000000E+00  0.000000000000E+00
 $END
 $OVERLAP
  0.100000000000E+01  0.241136651184E+00  0.100000000000E+01  0.000000000000E+00
  0.000000000000E+00  0.100000000000E+01  0.205480811879E-16  0.620947062815E-16
  0.000000000000E+00  0.100000000000E+01  0.623566895735E-17 -0.642262792663E-18
  0.000000000000E+00  0.198293399139E-32  0.100000000000E+01  0.771982236976E-01
  0.412771702639E+00  0.000000000000E+00 -0.525321457201E+00 -0.484293488405E-18
  0.100000000000E+01
 $END
 $DENSITY
  0.202403003928E+01 -0.905711149881E-01  0.405099750654E+00  0.443670199401E-15
 -0.167671587967E-15  0.151736136465E-30  0.133600109549E+00 -0.312415778650E+00
  0.142232362270E-15  0.242965510152E+00 -0.121427416572E-14 -0.372989996795E-15
 -0.108666243763E-30  0.246366769182E-15  0.118380577256E-29 -0.175530175032E+00
  0.490017210361E+00 -0.216192447563E-15 -0.380003691122E+00 -0.408451577516E-15
  0.594907304066E+00
 $END
 $FOCK
 -0.234464205046E+01 -0.639924565814E+00 -0.230204268148E+00 -0.756267297168E-16
  0.456720354123E-17  0.163837110032E+00 -0.137100147182E-01  0.109882176736E+00
 -0.297916896682E-16 -0.740238192171E-02  0.225599508703E-15  0.693781566163E-16
  0.119359460996E-31 -0.539243054439E-16  0.163837110032E+00 -0.210219105730E+00
 -0.279867759811E+00  0.370350939595E-16  0.239459985228E+00  0.924913406277E-16
 -0.163628013475E+00
 $END
 $LCAOMO
  0.991163558309E+00  0.322436589796E-01  0.189032539774E-15  0.689706086387E-02
 -0.676242951470E-15  0.613170748028E-02  0.172075042696E+00 -0.448898899289E+00
  0.200336634540E-15  0.348475516539E+00  0.366876487208E-15 -0.545358647311E+00
 -0.209100448505E+00  0.800766034469E+00  0.232220599236E-15  0.612716979294E+00
  0.126358439694E-15 -0.135475143059E+00 -0.127463846023E-15  0.407878460771E-15
 -0.930810908598E+00  0.360401774686E-15  0.365501097720E+00 -0.749844454576E-16
  0.316947086212E-16 -0.837005516475E-16  0.365501097720E+00 -0.174132053832E-15
  0.930810908598E+00  0.280414384775E-16  0.837582238042E-01 -0.731153863931E+00
  0.559497682997E-16  0.100100383332E+01 -0.753220754123E-16  0.122151352921E+01
 $END
 $KINETIC
  0.357678642449E+01 -0.202373805613E-01  0.102163366117E+00  0.000000000000E+00
  0.000000000000E+00  0.319681669225E+00 -0.460752772032E-16  0.980952679899E-18
  0.000000000000E+00  0.319681669225E+00 -0.151106301650E-17 -0.473339995719E-18
  0.000000000000E+00  0.110461960951E-33  0.319681669225E+00 -0.167562356894E-01
  0.469315040394E-01  0.000000000000E+00 -0.139729321775E+00  0.778725531922E-19
  0.760031883567E+00
 $END
 $NUCLEAR
 -0.832874881867E+01 -0.104773582571E+01 -0.150578768539E+01  0.000000000000E+00
  0.000000000000E+00 -0.146212120661E+01  0.175981856564E-01  0.125931970900E+00
  0.000000000000E+00 -0.156335787944E+01 -0.254976839521E-16  0.719139283033E-18
  0.000000000000E+00 -0.664457201313E-18 -0.146212120661E+01 -0.335239398727E+00
 -0.784550376331E+00  0.000000000000E+00  0.101277659836E+01  0.205453262948E-17
 -0.226052662359E+01
 $END
