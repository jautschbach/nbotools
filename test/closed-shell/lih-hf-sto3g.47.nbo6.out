
 *********************************** NBO 6.0 ***********************************
             N A T U R A L   A T O M I C   O R B I T A L   A N D
          N A T U R A L   B O N D   O R B I T A L   A N A L Y S I S
 *************************** Erik Van Lenthe (100424) **************************
  (c) Copyright 1996-2017 Board of Regents of the University of Wisconsin System
      on behalf of the Theoretical Chemistry Institute.  All rights reserved.

          Cite this program as:

          NBO 6.0.  E. D. Glendening, J. K. Badenhoop, A. E. Reed,
          J. E. Carpenter, J. A. Bohmann, C. M. Morales, C. R. Landis,
          and F. Weinhold (Theoretical Chemistry Institute, University
          of Wisconsin, Madison, WI, 2013); http://nbo6.chem.wisc.edu/

       /NAONHO / : Write the NAO to NHO transformation to lfn49
       /NHONBO / : Write the NHO to NBO transformation to lfn49
       /NBONLMO/ : Write the NBO to NLMO transformation to lfn49
       /NLMOMO / : Write the NLMO to MO transformation to lfn49
       /FAO    / : Write the AO Fock matrix to lfn49
       /SAO    / : Print the AO overlap matrix
       /SPNAO  / : Print the PNAO overlap matrix
       /SPNBO  / : Write the PNBO overlap matrix to lfn49
       /PLOT   / : Write information for the orbital plotter
       /FILE   / : Set to lih-hf-sto3g

 Filename set to lih-hf-sto3g

 Job title:  ORCA Job: lih-hf-sto3g


 AO overlap matrix:

          AO         1       2       3       4       5       6
      ---------- ------- ------- ------- ------- ------- -------
   1. Li 1 (s)    1.0000  0.2411  0.0000  0.0000  0.0000  0.0772
   2. Li 1 (s)    0.2411  1.0000  0.0000  0.0000 -0.0000  0.4128
   3. Li 1 (pz)   0.0000  0.0000  1.0000  0.0000  0.0000  0.0000
   4. Li 1 (px)   0.0000  0.0000  0.0000  1.0000  0.0000 -0.5253
   5. Li 1 (py)   0.0000 -0.0000  0.0000  0.0000  1.0000 -0.0000
   6.  H 2 (s)    0.0772  0.4128  0.0000 -0.5253 -0.0000  1.0000


 NATURAL POPULATIONS:  Natural atomic orbital occupancies

  NAO Atom No lang   Type(AO)    Occupancy      Energy
 -------------------------------------------------------
   1   Li  1  s      Cor( 1s)     1.99993      -2.34722
   2   Li  1  s      Val( 2s)     0.39561       0.06532
   3   Li  1  px     Ryd( 2p)     0.16209       0.37487
   4   Li  1  py     Ryd( 2p)     0.00000       0.16384
   5   Li  1  pz     Ryd( 2p)     0.00000       0.16384

   6    H  2  s      Val( 1s)     1.44237      -0.08599


 Summary of Natural Population Analysis:

                                     Natural Population
             Natural    ---------------------------------------------
  Atom No    Charge        Core      Valence    Rydberg      Total
 --------------------------------------------------------------------
   Li  1    0.44237      1.99993     0.39561    0.16209     2.55763
    H  2   -0.44237      0.00000     1.44237    0.00000     1.44237
 ====================================================================
 * Total * -0.00000      1.99993     1.83797    0.16209     4.00000

                                 Natural Population
 ---------------------------------------------------------
   Core                       1.99993 ( 99.9966% of    2)
   Valence                    1.83797 ( 91.8987% of    2)
   Natural Minimal Basis      3.83791 ( 95.9476% of    4)
   Natural Rydberg Basis      0.16209 (  4.0524% of    4)
 ---------------------------------------------------------

    Atom No         Natural Electron Configuration
 ----------------------------------------------------------------------------
     Li  1      [core]2s( 0.40)2p( 0.16)
      H  2            1s( 1.44)


 PNAO overlap matrix:

          NAO        1       2       3       4       5       6
      ---------- ------- ------- ------- ------- ------- -------
   1. Li 1( 1s )  1.0000 -0.0000  0.0000  0.0000  0.0000  0.0911
   2. Li 1( 2s ) -0.0000  1.0000  0.0000 -0.0000  0.0000  0.4033
   3. Li 1( 2px)  0.0000  0.0000  1.0000  0.0000  0.0000 -0.5253
   4. Li 1( 2py)  0.0000 -0.0000  0.0000  1.0000  0.0000 -0.0000
   5. Li 1( 2pz)  0.0000  0.0000  0.0000  0.0000  1.0000  0.0000
   6.  H 2( 1s )  0.0911  0.4033 -0.5253 -0.0000  0.0000  1.0000


 NATURAL BOND ORBITAL ANALYSIS:

                            Occupancies       Lewis Structure    Low   High
         Max    Occ     -------------------  -----------------   occ   occ
  Cycle  Ctr   Thresh    Lewis   non-Lewis     CR  BD  nC  LP    (L)   (NL)
 ============================================================================
    1     2     1.90     3.99993   0.00007      1   1   0   0     0      0
 ----------------------------------------------------------------------------

 Structure accepted: No low occupancy Lewis orbitals

 -------------------------------------------------------
   Core                      1.99993 ( 99.997% of   2)
   Valence Lewis             2.00000 (100.000% of   2)
  ==================      =============================
   Total Lewis               3.99993 ( 99.998% of   4)
  -----------------------------------------------------
   Valence non-Lewis         0.00003 (  0.001% of   4)
   Rydberg non-Lewis         0.00004 (  0.001% of   4)
  ==================      =============================
   Total non-Lewis           0.00007 (  0.002% of   4)
 -------------------------------------------------------


     (Occupancy)   Bond orbital / Coefficients / Hybrids
 ------------------ Lewis ------------------------------------------------------
   1. (1.99993) CR ( 1)Li  1            s(100.00%)
                                         1.0000  0.0000  0.0000  0.0000  0.0000
   2. (2.00000) BD ( 1)Li  1- H  2
               ( 27.88%)   0.5280*Li  1 s( 70.94%)p 0.41( 29.06%)
                                         0.0000  0.8423 -0.5390 -0.0000 -0.0000
               ( 72.12%)   0.8492* H  2 s(100.00%)
                                         1.0000
 ---------------- non-Lewis ----------------------------------------------------
   3. (0.00003) BD*( 1)Li  1- H  2
               ( 72.12%)   0.8492*Li  1 s( 70.94%)p 0.41( 29.06%)
               ( 27.88%)  -0.5280* H  2 s(100.00%)
   4. (0.00004) RY ( 1)Li  1            s( 29.06%)p 2.44( 70.94%)
   5. (0.00000) RY ( 2)Li  1            s(  0.00%)p 1.00(100.00%)
   6. (0.00000) RY ( 3)Li  1            s(  0.00%)p 1.00(100.00%)


 NHO DIRECTIONALITY AND BOND BENDING (deviation from line of nuclear centers at
                                      the position of maximum hybrid amplitude)

         [Thresholds for printing:  angular deviation  >  1.0 degree]
                                    p- or d-character  > 25.0%
                                    orbital occupancy  >  0.10e

                        Line of Centers        Hybrid 1             Hybrid 2
                        ---------------  -------------------  ------------------
           NBO            Theta   Phi    Theta   Phi    Dev   Theta   Phi    Dev
 ===============================================================================
 Failed to locate density maximum for hybrid 2 on center 1


 SECOND ORDER PERTURBATION THEORY ANALYSIS OF FOCK MATRIX IN NBO BASIS

     Threshold for printing:   0.50 kcal/mol
                                                          E(2) E(NL)-E(L) F(L,NL)
      Donor (L) NBO              Acceptor (NL) NBO      kcal/mol   a.u.    a.u.
 ===============================================================================

 within unit  1
       None above threshold


 NATURAL BOND ORBITALS (Summary):

                                                     Principal Delocalizations
           NBO                 Occupancy    Energy   (geminal,vicinal,remote)
 ===============================================================================
 Molecular unit  1  (HLi)
 ------ Lewis --------------------------------------
    1. CR ( 1)Li  1             1.99993    -2.34722
    2. BD ( 1)Li  1- H  2       2.00000    -0.29113
 ------ non-Lewis ----------------------------------
    3. BD*( 1)Li  1- H  2       0.00003     0.44459
    4. RY ( 1)Li  1             0.00004     0.20074
    5. RY ( 2)Li  1             0.00000     0.16384
    6. RY ( 3)Li  1             0.00000     0.16384
          -------------------------------
                 Total Lewis    3.99993  ( 99.9983%)
           Valence non-Lewis    0.00003  (  0.0008%)
           Rydberg non-Lewis    0.00004  (  0.0009%)
          -------------------------------
               Total unit  1    4.00000  (100.0000%)
              Charge unit  1    0.00000

 $CHOOSE
   BOND S 1 2 END
 $END


 NATURAL LOCALIZED MOLECULAR ORBITAL (NLMO) ANALYSIS:

 1 pass over DM reduced the maximum off-diagonal element to 0.34E-12

 Hybridization/Polarization Analysis of NLMOs in NAO Basis:
 NLMO / Occupancy / Percent from Parent NBO / Atomic Hybrid Contributions
 -------------------------------------------------------------------------------
   1. (2.00000)  99.9966% CR ( 1)Li  1
                         100.000% Li  1 s(100.00%)p 0.00(  0.00%)
   2. (2.00000) 100.0000% BD ( 1)Li  1- H  2
                          27.882% Li  1 s( 70.94%)p 0.41( 29.06%)
                          72.118%  H  2 s(100.00%)

 NORMAL TERMINATION of nbo program

 Maximum scratch memory used by NBO was 130817 words (1.00 MB)
 Maximum scratch memory used by GENNBO was 138 words (0.00 MB)
